common/djangoapps/third_party_auth/models.py

```diff
@@ -347,6 +347,9 @@ class SAMLProviderConfig(ProviderConfig):
     attr_email = models.CharField(
         max_length=128, blank=True, verbose_name="Email Attribute",
         help_text="URN of SAML attribute containing the user's email address[es]. Leave blank for default.")
+    include_in_saml_pull = models.BooleanField(
```

Comment:
I'm okay with renaming the field as such -- I agree that this should not be used as a "silence" button.  My original intent was to use the expiration date field instead of adding a new switch, but @bradenmacdonald was concerned that it would prevent IdPs from being able to update their metadata on-the-fly when needed -- which this flag would also do, but at least it would be explicit on our end.